src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,28 +46,25 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING,
+    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The left padding of the text container has been doubled. If this is intentional and required for your design, it's fine. However, if you meant to add some other padding value, please correct it.
```diff
- paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
+ paddingLeft: BUBBLE_LEFT_PADDING + SOME_OTHER_PADDING_VALUE,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->